src/main/java/com/meetme/plugins/jira/gerrit/tabpanel/SubtaskReviewsTabPanel.java

```diff
@@ -81,4 +86,12 @@ private boolean isConfigurationReady() {
         return configuration != null && configuration.getSshHostname() != null && configuration.getSshUsername() != null
                 && configuration.getSshPrivateKey() != null && configuration.getSshPrivateKey().exists();
     }
+
```

Comment:
Seems like duplicated code. What about adding `configuration ` as argument and making the method static?
Another option is to add it as a default method to `GerritConfiguration`.

Reply:
Thanks for found it. I hope, the better review interface and set it as static method. 
Now as workaround I have migrated into imp class, and define method in interface. 
Yes, it is old way, but it does not duplicate of code :)